plone/app/locales/locales/it/LC_MESSAGES/plone.po

```diff
@@ -535,7 +535,7 @@ msgstr "Aggiungi una condizione TALES"
 
 #: plone.app.contentrules/plone/app/contentrules/actions/versioning.py:71
 msgid "Add Versioning Action"
-msgstr ""
+msgstr "Aggiungi Azione di Versionamento"
```

Comment:
Nelle altre traduzioni "Azione" ha la "a" minuscola, la metterei minuscola per coerenza.

Reply:
Per coerenza sarei d'accordo ma concettualmente una "Action" è qualcosa di specifico in Plone... forse sarebbe da fare il cambio opposto :|